mbuild/compound.py

```diff
@@ -1138,7 +1138,159 @@ def _kick(self):
             particle.pos += (np.random.rand(3,) - 0.5) / 100
         self._update_port_locations(xyz_init)
 
-    def energy_minimization(self, steps=2500, algorithm='cg',
+    def energy_minimization(self, forcefield='UFF',steps=1000, ):
```

Comment:
I've been thinking for awhile that it makes more sense for this method to be called `energy_minimize` instead of `energy_minimization`. Let's go ahead and rename this, but also keep a method for `energy_minimization` that is deprecated (see [here](https://github.com/mosdef-hub/mbuild/blob/master/mbuild/coordinate_transform.py#L397) for usage of the decorator we have to denote this). For now `energy_minimization` can just call `energy_minimize`.